Libraries/Components/TextInput/TextInput.js

```diff
@@ -287,6 +287,27 @@ var TextInput = React.createClass({
      */
     style: Text.propTypes.style,
     /**
+     * Used to set whether this view is accessible
+     */
+    accessible: PropTypes.bool,
+    /**
+     * Used to set the accessibility label of this view
+     */
+    accessibilityLabel: PropTypes.string,
+    /**
+     * Used to set the accessibilityTraits of this view
+     * @platform ios
+     */
+    accessibilityTraits: React.PropTypes.oneOfType([
+          React.PropTypes.oneOf(View.AccessibilityTraits),
+          React.PropTypes.arrayOf(React.PropTypes.oneOf(View.AccessibilityTraits)),
+        ]),
+    /**
+     * Used to set the accessibility component type of this view
+     * @platform android
+     */
+    accessibilityComponentType: React.PropTypes.oneOf(View.AccessibilityComponentType),
+    /**
```

Comment:
All of these should be already pulled in via the `...View.propTypes` above (see `View.js`).